src/main.ts

```diff
@@ -3,6 +3,6 @@ import { AppModule } from './app.module';
 
 async function bootstrap() {
   const app = await NestFactory.create(AppModule);
-  await app.listen(3000);
+  await app.listen(8080);
```

Comment:
Does the 8080 Port have any special uses for the App Platform? 
What happens if we use  3000 port?